standard/tier/tier_utils.lua

```diff
@@ -0,0 +1,177 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Tier/Utils
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+-- module intended to be moved to `Module:Tier` after the old ones usage has been eliminated
+
+local HiddenSort = require('Module:HiddenSort')
+local Logic = require('Module:Logic')
+local Page = require('Module:Page')
+local String = require('Module:StringUtils')
+
+local TierData = mw.loadData('Module:Tier/Data')
+
+local NON_BREAKING_SPACE = '&nbsp;'
+
+local Tier = {}
+
+--- Converts input to standardized identifier format
+---@param input string|integer|nil
+---@return string|integer|nil
+function Tier.toIdentifier(input)
```

Comment:
Maybe `Tier.read(input)`?

Reply:
imo read would rather be something like
`Tier.value(Tier.toIdentifier(input), mode)`

Reply:
What precisely are the valid tiers? (assuming commons)

Reply:
the ones listed in the data module

Reply:
Is it `1/2/3/4/5`? or `'1'/'2'/'3'/'4/'5'`? Is nil a valid tier? what about `''`?

Reply:
for tier
1,2,3,4,5 and as fallback '' are valid
via the toIdentifier '1','2','3','4','5' and nil are made to its valid input representative

Reply:
ok thank you